admin/media_types.php

```diff
@@ -10,173 +10,181 @@
-                $media_types = $db->Execute($media_type_query_raw);
-                foreach ($media_types as $media_type) {
-                  if ((!isset($_GET['mID']) || (isset($_GET['mID']) && ($_GET['mID'] == $media_type['type_id']))) && !isset($mInfo) && (substr($action, 0, 3) != 'new')) {
-                    $mInfo = new objectInfo($media_type);
-                  }
-
-                  if (isset($mInfo) && is_object($mInfo) && ($media_type['type_id'] == $mInfo->type_id)) {
+                    $media_type_split = new splitPageResults($_GET['page'], MAX_DISPLAY_SEARCH_RESULTS, $media_type_query_raw, $media_type_query_numrows);
+                    $media_types = $db->Execute($media_type_query_raw);
+                    foreach ($media_types as $media_type) {
+                        if ((!isset($_GET['mID']) || (isset($_GET['mID']) && ($_GET['mID'] == $media_type['type_id']))) && !isset($mInfo) && (substr($action, 0, 3) != 'new')) {
+                            $mInfo = new objectInfo($media_type);
+                        }
+
+                        if (isset($mInfo) && is_object($mInfo) && ($media_type['type_id'] == $mInfo->type_id)) {
+                            ?>
+                            <tr id="defaultSelected" class="dataTableRowSelected" onclick="document.location.href='<?php echo zen_href_link(FILENAME_MEDIA_TYPES, 'page=' . $_GET['page'] . '&mID=' . $media_type['type_id'] . '&action=edit'); ?>'">
+                        <?php } else { ?>
+                            <tr class="dataTableRow" onclick="document.location.href='<?php echo zen_href_link(FILENAME_MEDIA_TYPES, 'page=' . $_GET['page'] . '&mID=' . $media_type['type_id'] . '&action=edit'); ?>'">
+                        <?php } ?>
+                        <td class="dataTableContent"><?php echo $media_type['type_name']; ?></td>
+                        <td class="dataTableContent"><?php echo $media_type['type_ext']; ?></td>
+                        <td class="dataTableContent text-right">
+                            <?php
+                            if (isset($mInfo) && is_object($mInfo) && ($media_type['type_id'] == $mInfo->type_id)) {
+                                echo zen_image(DIR_WS_IMAGES . 'icon_arrow_right.gif', '');
+                            } else {
+                                echo '<a href="' . zen_href_link(FILENAME_MEDIA_TYPES, zen_get_all_get_params(['mID']) . 'mID=' . $media_type['type_id']) . '">' . zen_image(DIR_WS_IMAGES . 'icon_info.gif', IMAGE_ICON_INFO) . '</a>';
+                            }
+                            ?>
+                        </td>
+                        </tr>
+                        <?php
+                    }
+                    $mType_parameter = '';
+                    if (!$media_types->EOF) {
+                        $mType_parameter = '&mID=' . $mInfo->type_id;
```

Comment:
Is it possible that `$mInfo` still could be undefined here?  
Ref line 87

Reply:
i really do not think so; but i will adjust the code to reflect that possibility.